Source/ui_unix/openglwindow.cpp

```diff
@@ -0,0 +1,34 @@
+#include "openglwindow.h"
+#include <QResizeEvent>
+
+OpenGLWindow::OpenGLWindow(QWindow *parent)
+    : QWindow(parent)
+{
+    QSurfaceFormat format;
+    format.setVersion(3, 2);
+    format.setProfile(QSurfaceFormat::CompatibilityProfile);
```

Comment:
Humm, just tested further and it might not be as trivial as changing this to `CoreProfile`. I've tried removing the `glewExperimental` line (which shouldn't be there), but it crashes a bit further down the road trying to call `glClearDepthf`. From what I can see, it's only supported on OpenGL 4.1+ which might explain the crash. I'll see what's the best course of action for that.